netmiko/hp/hp_comware.py

```diff
@@ -34,21 +34,25 @@ def session_preparation(self):
         time.sleep(0.3 * self.global_delay_factor)
         self.clear_buffer()
 
-    def config_mode(self, config_command="system-view"):
+    def config_mode(
+        self, config_command: str = "system-view", pattern: str = "", re_flags: int = 0
+    ) -> str:
         """Enter configuration mode."""
-        return super().config_mode(config_command=config_command)
+        return super().config_mode(
+            config_command=config_command, pattern=pattern, re_flags=re_flags
+        )
 
-    def exit_config_mode(self, exit_config="return", pattern=r">"):
+    def exit_config_mode(self, exit_config="return", pattern=r">") -> str:
```

Comment:
Your missing the type hints on the arguments i.e. `exit_config: str = "return"` et cetera.